packages/server/tests/postman.json

```diff
@@ -0,0 +1,820 @@
+			  "    pm.expect(response.jsonrpc).to.equal(\"2.0\");",
+			  "});",
+			  ""
+			],
+			"type": "text/javascript"
+		  }
+		}
+	  ],
+	  "request": {
+		"method": "POST",
+		"header": [],
+		"body": {
+		  "mode": "raw",
+		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"eth_syncing\",\n    \"params\": []\n}",
+		  "options": {
+			"raw": {
+			  "language": "json"
+			}
+		  }
+		},
+		"url": {
+		  "raw": "{{base_url}}",
+		  "host": [
+			"{{base_url}}"
+		  ]
+		}
+	  },
+	  "response": []
+	},
+	{
+	  "name": "net_listening",
+	  "event": [
+		{
+		  "listen": "test",
+		  "script": {
+			"exec": [
+			  "pm.test(\"Success\", () => {",
+			  "    var response = pm.response.json();",
+			  "    pm.expect(response.result).to.equal(\"false\");",
+			  "    pm.expect(response.id).to.equal(\"test_id\");",
+			  "    pm.expect(response.jsonrpc).to.equal(\"2.0\");",
+			  "});",
+			  ""
+			],
+			"type": "text/javascript"
+		  }
+		}
+	  ],
+	  "request": {
+		"method": "POST",
+		"header": [],
+		"body": {
+		  "mode": "raw",
+		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"net_listening\",\n    \"params\": []\n}",
+		  "options": {
+			"raw": {
+			  "language": "json"
+			}
+		  }
+		},
+		"url": {
+		  "raw": "{{base_url}}",
+		  "host": [
+			"{{base_url}}"
+		  ]
+		}
+	  },
+	  "response": []
+	},
+	{
+	  "name": "net_version",
+	  "event": [
+		{
+		  "listen": "test",
+		  "script": {
+			"exec": [
+			  "pm.test(\"Success\", () => {",
+			  "    var response = pm.response.json();",
+			  "    pm.expect(response.result).to.match(/^0x[a-f0-9]+$/);",
+			  "    pm.expect(response.id).to.equal(\"test_id\");",
+			  "    pm.expect(response.jsonrpc).to.equal(\"2.0\");",
+			  "});",
+			  ""
+			],
+			"type": "text/javascript"
+		  }
+		}
+	  ],
+	  "request": {
+		"method": "POST",
+		"header": [],
+		"body": {
+		  "mode": "raw",
+		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"net_version\",\n    \"params\": []\n}",
+		  "options": {
+			"raw": {
+			  "language": "json"
+			}
+		  }
+		},
+		"url": {
+		  "raw": "{{base_url}}",
+		  "host": [
+			"{{base_url}}"
+		  ]
+		}
+	  },
+	  "response": []
+	},
+	{
+	  "name": "web3_clientVersion",
+	  "event": [
+		{
+		  "listen": "test",
+		  "script": {
+			"exec": [
+			  "pm.test(\"Success\", () => {",
+			  "    var response = pm.response.json();",
+			  "    pm.expect(response.result).to.match(/^relay[/]/);",
+			  "    pm.expect(response.id).to.equal(\"test_id\");",
+			  "    pm.expect(response.jsonrpc).to.equal(\"2.0\");",
+			  "});",
+			  ""
+			],
+			"type": "text/javascript"
+		  }
+		}
+	  ],
+	  "request": {
+		"method": "POST",
+		"header": [],
+		"body": {
+		  "mode": "raw",
+		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"web3_clientVersion\",\n    \"params\": []\n}",
```

Comment:
Still missing a case where the method called is not applicable
In the example postman.json I linked we did this by calling a non existent method e.g.
```
		  "raw": "{\n    \"id\": \"test_id\",\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"unknown\",\n    \"params\": []\n}",
```